f1tenth_racecar/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>f1tenth_racecar</name>
+  <version>0.0.0</version>
+  <description>The f1tenth_racecar package</description>
+  <maintainer email="whitleysoftwareservices@gmail.com">Josh Whitley</maintainer>
+  <license>BSD</license>
+  <url type="bugtracker">http://github.com/f1tenth/f1tenth_system/issues</url>
+  <url type="repository">http://github.com/f1tenth/f1tenth_system</url>
+  <author email="whitleysoftwareservices@gmail.com">Josh Whitley</author>
+  <author>Michael Boulet</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>actionlib</depend>
+  <depend>rospy</depend>
+  <depend>rosservice</depend>
+  <depend>rostopic</depend>
+  <depend>sensor_msgs</depend>
+
+  <exec_depend>ackermann_cmd_mux</exec_depend>
+  <exec_depend>amcl</exec_depend>
```

Comment:
acml, hector_mapping, robot_pose_ekf, are probably no longer needed.

Reply:
Removed unused dependencies after removing the above launch file.